src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -52,16 +53,61 @@ Hegel goes to add a link between the empty thinking (or empty intuiting) that fo
 
 Instead, it is better to understand Hegel's passage here to be an aid to the reader that merely reflects on the logic rather than developing it further. This being that the two categories appear to be equally devoid of any meaning, or map on to similar, if not same, manner of thinking. However, the claim that `being` is `nothing` is not on account of empty thinking or any other mental operation no, but purely through its concept.
 
+{/* prettier-ignore */}
 ```md
-&ndash; Nothing is therefore the same determination
-or absence of determination, and thus altogether
+- Nothing is therefore the same determination
+or absence of determination, and thus altogether 
 the same as what pure _being_ is.
 ```
 
 The final sentence begins with an en dash that may indicate the termination of the inlined comment and a return to the logical development. Further to this is the fact that the passage here has no superfluous material.
 
 It is this on account of the same determination or total lack of determination, which renders `nothing` to be the _same_ as `being`.
 
-Note that Hegel at this point does not strictly state that pure `nothing` is pure `being`, but only that `nothing ` is _the same as_ `being`. This may imply that `nothing` is `being` or that one has transitioned into the other, as Hegel goes on to indicate in the next category, but the text does not state this identification or transition explicitly.
+Note that Hegel at this point does not strictly state that pure `nothing` is pure `being`, but only that `nothing ` is _the same as_ `being`. This may still imply that `nothing` is `being` or that one has transitioned into the other, as Hegel goes on to indicate in the next category, but the present text does not state this identification or transition explicitly.
 
-{/* TODO note the difference between _is_ (as transition) and _same_ when thinking about being and nothing. former signifies immediate transition whereas latter seems to construe some essence, an "area" where being and nothing are con-joined. */}
+## Further Commentary
+
+### Burbidge
+
+John Burbidge's more psychological reading of the development of the _Logic_ offers helpful guidance in understanding the otherwise very minimal and obscure notion of `nothing`. He begins by stressing that the understanding or cognition that thinks this category has only `pure being` to compare `nothing` with and that it struggles to find any distinguishing identifier: each category is likewise "simply equal with itself", "complete emptiness, absence of all determination and content" and "internally undifferentiated or pure". Yet, some difference must remain, according to Burbidge:
+
+> The only difference that could possible remain is that _being_ represents that feature which _every_ thing has, and which enables it to be thought at all, whereas nothing is _no_ thing (Burbidge 1981, 39).
```

Comment:
I think you should come down a bit harder on Burbidge here. He's just smuggled in this idea of "every thing" and "no thing" and it's just completely unjustified. 
As I'm writing this, I reckon this is something that I could add to this article once it's been published.

Reply:
I add some criticism at the end of the Burbidge section. Given that this isn't a named critique - I am not critiquing in my own voice even though I'm writing this - the critique has to be quite general, obvious and made in a suggestive manner. E.g. I don't think I can use "To my mind, Burdidge is wrong here..." etc. kind of language because this is, in principle, a collaborative piece of writing.

Reply:
You raise an interesting point about how we should include secondary commentaries in the Reference section. I think you're right that we should probably just represent views rather than critique views within the Reference section. This is something that should be codified in practice. 

If the above is the case, then we will need to have dedicated sections to secondary lit. Maybe we can have Folders with Secondary authors and within each folder a list of files of particular disagreements?

Reply:
We can do that as the need arises, and it's an easy job to do so. It depends on the article. Here my plan was to include 3 commentaries and it seemed natural to bundle them together. If I make any comments in my own voice, I add my lastname in paranthesis to the section to indicate that.